src/core/less/structure/block.less

```diff
@@ -0,0 +1,39 @@
+// Block styling
+// =============
+
+.block {
+    background-color: @background-color;
+    position: relative;
+
+    .block-inner {
+        max-width: @device-width-large;
+        margin: auto;
+        padding: @block-padding-top 0 @block-padding-bottom;
+        // reduce padding as screen width decreases
+        @media all and (max-width: (@device-width-large + 20px)) {
```

Comment:
all of the media queries like the ones in this file are additional styling for a specific theme specific look, and should be removed

Reply:
Done.